critiquebrainz/db/users.py

```diff
@@ -0,0 +1,52 @@
+from hashlib import md5
+import sqlalchemy
+from critiquebrainz import db
+
+
+def avatar(user):
+    """Args:
+        user: user object from db (dict) or username string
```

Comment:
While `dict` is technically an object, it might be confusing to people. Just say that it requires a dictionary with user data. Also specify exactly which data is required.